helm/mds/tests/auth-agency-jwt-audience_test.yaml

```diff
@@ -0,0 +1,73 @@
+suite: test authetication jwt audience agency
+templates:
+  - auth.yaml
+tests:
+  - it: doc 1
+    release:
+      namespace: mds
+    set:
+      apis:
+        - name: mds-provider
+        - name: mds-agency
+          pathPrefix: /agency
```

Comment:
Why is `pathPrefix` specified for `mds-agency` but not for other APIs?

Reply:
these is a configuration overrlay/override whereby we are only testing a conditional agency deployment and as such only need to override the mds-agency configuration data, and because the apis are structured as a list, vs keyed map, we have to include the entire list when doing so, even stubs :(